packages/gerber-parser/test/gerber-parser_gerber_test.js

```diff
@@ -162,6 +162,28 @@ describe('gerber parser with gerber files', function() {
       p.write('%MOMM*%\n')
     })
 
+    // Special Case: Cadence Allegro
+    // The gerber spec says
+    //   "There can be only one extended code command between each pair of ‘%’ delimiters"
+    // Cadence Allegro violates this rule by including the units after the format command
+    //
+    // Sample:
+    //   G04 File Origin:  Cadence Allegro 17.2-S032*
+    //   ...
+    //   %FSLAX25Y25*MOMM*%
+    //   ...
+    //
+    it('should set units if found in same line as format', function(done) {
+      var expected = [
```

Comment:
Can you expand this test to make sure the format spec is also parsed properly?

Reply:
Happy to. No worries on the delay, i'm happy you go around to it!

I'm not certain how many tests to add or which describe to slot them under. For now i'll assume are few more expects in this tests will suffice. If you have something more in mind lmk

Reply:
there was def a bug there. nota and epsilion were not being output if the units were detected.

I needed to create 2 tests: one using expectResults (to check outputs), and one without (for property checks). Didn't see a cleaner approach.

If more tweaks are needed pls lmk.